roles/yoda_zabbix_system/templates/zabbix_agentd.conf.j2

```diff
@@ -235,7 +235,7 @@ HostnameItem=system.hostname
 # Mandatory: no
 # Range: 1-30
 # Default:
-# Timeout=3
```

Comment:
For an administrator it is useful to be able to see in the processed file what the original default was. The other lines where an ansible variable is filled in also use that pattern. So I'd advise to leave the "# Timeout=3" line in.